.github/workflows/pr.yml

```diff
@@ -0,0 +1,14 @@
+name: pr
+on:
+  pull_request:
```

Comment:
Shouldn't this be `pull_request_target`? The docs seem to imply that `pull_request` should only be used temporarily for testing upgrades…

Reply:
We use `pull_request` in the build.yml file, since `pull_request_target` is runs in the base branch context. `pull_request_target` also has *interesting* security implications.

I don't think this matters for here anyways, but I can change it if you see how this might break

Reply:
I'm not sure what the exact implications are TBH, just noticed the deviation from the docs. I'm completely OK with just leaving this as-is and wait-and-see if there are any issues with it.